docker-compose.yml

```diff
@@ -34,7 +34,7 @@ services:
       timeout: 5s
       retries: 5
     volumes:
-      - committeeoversight-db-data:/var/lib/postgresql/
+      - committeeoversight-db-data:/var/lib/postgresql/data
```

Comment:
i updated the volume mount because our data was not persisting between container runs, because we were not mounting it onto the directory in which postgres stores db data. see https://hub.docker.com/_/postgres/#pgdata!

Reply:
🙏🏻